### PR TITLE
NEPT-1132: Component ID 58 - Implements the select boxes component

### PR DESCRIPTION
**Pull request by drupol**

### CODE REVIEW

File: includes/alter/form.inc [hunk 1]
```diff
@@ -19,170 +19,6 @@ function ec_europa_form_alter(&$form, &$form_state, $form_id) {
...
-    && drupal_valid_path($element['#autocomplete_path'])
-  ) {
-    $attributes['class'][] = 'form-autocomplete';
-  }
-
-  $attributes['class'][] = 'form-item';
-
-  // See https://www.drupal.org/node/154137
-  if (!empty($element['#id']) && $element['#id'] == 'edit-querytext') {
-    $element['#children'] = str_replace('type="text"', 'type="search"', $element['#children']);
```
Comment (drupol) [inline includes/alter/form.inc]: ? what ?

Comment (gillesdeudon) [inline includes/alter/form.inc]: Just a comment on the history nothing to do with your job.

Comment (drupol) [inline includes/alter/form.inc]: 👍

File: templates/select/select.component.inc [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    ),
+  );
+}
+
+/**
+ * Implements hook_preprocess_hook().
+ */
+function ec_europa_preprocess_select(&$variables, $hook) {
+  $variables['attributes_array']['class'][] = 'ecl-select';
+  if (in_array('error', $variables['attributes_array']['class'])) {
```
Comment (drupol) [inline templates/select/select.component.inc]: @gillesdeudon that function is not yet available in this branch.

I propose to revisit this once both tickets are merged.

Comment (gillesdeudon) [inline templates/select/select.component.inc]: I agree. I will do it in nept-1131, once it is done.

I will comment in in nept-1132 and nept-1131.

**Comment by gillesdeudon** [comment]
👍

**Comment by voidtek** [comment]
:+1:

**Comment by gillesdeudon** [comment]
👍

**Comment by drupol** [comment]
Thanks !

**Pull request merged**